about.html

```diff
@@ -10,64 +10,106 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<!----header-->
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="header">
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <div class="logo-links-container">
+        
+            <nav>
+              <img src="img/lambda-black.png">
+              <a href="#">Home</a>
+              <a href="#">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+            </nav>
+        
+    </div>
 
+        <img src="img/jumbo-about.png">
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+     <div class="text-container">
+
+        <h1>About</h1>
 
-    Learn More
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
 
+     
+    </div>
+  </div>
 
+     <!-----CTA-->
+      <div class="block-container">
 
-    <img src="img/about-working.png" alt="strategy">
+        <div class="block">
+            <img src="img/about-plan.png" alt="strategy">
     
-    How We Work
+            <h1>Strategy</h1>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+            <button href="#">Learn More</button>
+        </div>  
 
```

Comment:
The only thing it looks like you are missing are the dashed bottom border lines on each of these 4 sections.